server/utils/validation.test.ts

```diff
@@ -180,3 +185,53 @@ describe('fetchErrorsAndUserInput', () => {
     expect(result).toEqual({ errors, errorSummary, userInput })
   })
 })
+
+describe('insertPropertyError', () => {
```

Comment:
Maybe `insertGenericError` would be better, as we already have the concept of `Properties`